src/hls-adapter.js

```diff
@@ -184,15 +197,17 @@ export default class HlsAdapter extends BaseMediaSourceAdapter {
       if (abr.defaultBandwidthEstimate) {
         adapterConfig.hlsConfig.abrEwmaDefaultEstimate = abr.defaultBandwidthEstimate;
       }
-
       if (abr.restrictions) {
-        if (abr.restrictions.minBitrate > 0) {
-          adapterConfig.hlsConfig.minAutoBitrate = abr.restrictions.minBitrate;
-        }
-        if (abr.restrictions.maxBitrate < Infinity) {
-          //You can either set capping by size or bitrate, if bitrate is set then disable size capping
-          adapterConfig.hlsConfig.capLevelToPlayerSize = false;
-          adapterConfig.abr.restrictions = abr.restrictions;
+        if (abr.restrictions.maxBitrate > abr.restrictions.minBitrate) {
```

Comment:
couldn't be equal for a specific one? anyway add it to the previous one

Reply:
That's what we had before but it's correct